### PR TITLE
fix margins

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam after remove metada in public view we need fix some margins, could you take a look?

### CODE REVIEW

File: themes/scss/widgets/filter.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 .CDB-Widget-filter {
   display: block;
   max-height: 200px;
-  margin: 5px 0 $baseSize;
```
Comment (xavijam) [inline themes/scss/widgets/filter.scss]: no basesize?

Comment (piensaenpixel) [inline themes/scss/widgets/filter.scss]: you're right

**Comment by xavijam** [comment]
:+1:

File: themes/scss/widgets/filter.scss (hunk 1 above)
Comment (piensaenpixel) [inline themes/scss/widgets/filter.scss]: Fixed!

**Pull request merged**